dev/PipsPager/PipsPager.idl

```diff
@@ -10,15 +10,15 @@ enum PipsPagerButtonVisibility
     Collapsed
 };
 
-[WUXC_VERSION_PREVIEW]
+[MUX_PREVIEW]
```

Comment:
This might conflict with Seva's PR.

Reply:
Sure, I can wait until his change is in and merge. The work I need to do is in preparation of post-2.6.